src/test/java/com/ft/methodearticlemapper/transformation/BodyProcessingFieldTransformerFactoryTest.java

```diff
@@ -1204,6 +1204,14 @@ public void shouldKeepCiteTagInsideBlockquote() {
 		checkTransformation(originalContent, transformedContent);
 	}
 
+	@Test
+	public void shouldKeepAsideTag(){
+		String originalContent = "<body><p>Here some text</p><aside>Aside keep<h2>This is the title</h2><p>This is the description</p><p>blah blah blah</p></aside></body>";
```

Comment:
There is no title and description in an aside, not defined anyways. Changed that inner text to something else, as of now it looks like the podcast-promo.